rpa-bot/1.3/modules/ROOT/pages/_partials/hard-soft-requirements.adoc

```diff
@@ -28,12 +30,28 @@ include::rpa-home::partial$windows-versions-support.adoc[]
 
 |===
 
-=== Requirements for Secure Session Technology
+=== System Settings
+
+* Display resolution: 
+** Minimum: 1280 x 1080
+** Recommended: Full HD 1080p (1920 × 1080) or higher
+* Scaling of text and other items on the screen: 100 %
+* _Clear Type_: turned off
+
+=== Requirements for the Secure Session Technology
+
+Secure Sessions are hidden Desktop sessions. They run in the background and only the bot can access them. An additional advantage of using Secure Sessions is, that the bot can multiply itsself to run several process instances in parallel.
+
+To use Secure Sessions you must ensure that all prerequisites are met and all responsible persons are informed.
 
-* Windows logins used for secure sessions have privileges to open a local RDP session on `127.0.0.2`.
-* 64-bit Operating System.
-* Dedicated Windows Server machine with enough resources to execute several processes simultaneously (2 GB or more RAM per session)
-* The RPA Bot service runs under the _Local System account_ (by default, the setup configures RPA Bot to use this account).
+* The RPA Bot service runs under the _Local System_ account per default.
```

Comment:
Why italics?

Reply:
It is a name used by Windows, see other comment

Reply:
replaced with monospaced, see Slack